geode-core/src/main/java/org/apache/geode/internal/cache/PartitionedRegionDataStore.java

```diff
@@ -1257,7 +1257,7 @@ boolean canAccommodateMoreBytesSafely(int bytes) {
     final long curBytes = bytesInUse.get();
     if (isDebugEnabled) {
       logger.debug(
-          "canAccomodateMoreBytes: bytes = {} allocatedMemory = {} newAllocatedSize = {} thresholdSize = ",
+          "canAccomodateMoreBytes: bytes ={} allocatedMemory ={} newAllocatedSize ={} thresholdSize ={}",
```

Comment:
Was it intentional to remove the whitespace here?

Reply:
was trying to keep the spacing unform in a way. There are no spaces in ':{}' (sometimes they do) hence was trying to keep things uniform.

Reply:
If removing whitespace, `x=5 y=7` seems more natural than `x =5 y =7` 🤷‍♂️